mev_inspect/crud/classified_traces.py

```diff
@@ -22,9 +22,30 @@ def write_classified_traces(
     db_session,
     classified_traces: List[ClassifiedTrace],
 ) -> None:
-    models = [
-        ClassifiedTraceModel(**json.loads(trace.json())) for trace in classified_traces
-    ]
+    models = []
+    for trace in classified_traces:
+        models.append(
+            ClassifiedTraceModel(
+                **{
```

Comment:
the spread (**) is for cases where you have a dict and want to assign the keys to the args of a function.

Since you explicitly know the argument names, better to do 
```
ClassifiedTraceModel(
    transaction_hash=trace.transaction_hash,
    block_number=trace.block_number
    ...
```